src/Popover/Popover.js

```diff
@@ -79,22 +79,22 @@ const Popover = React.createClass({
 
     /**
      * This is the point on the popover which will stick to
-     * the anchors origin.
+     * the anchor's origin.
      * Options:
-     * vertical: [top, middle, bottom]
-     * horizontal: [left, center, right]
+     * vertical: [top, middle, bottom];
+     * horizontal: [left, center, right].
      */
     targetOrigin: propTypes.origin,
 
     /**
      * If true, the popover will render on top of an invisible
      * layer, which will prevent clicks to the underlying
-     * elements, and trigger an onRequestClose(clickAway) event.
+     * elements, and trigger an `onRequestClose('clickAway')` call.
      */
     useLayerForClickAway: React.PropTypes.bool,
 
     /**
-     * This number represents the zDepth of the paper shadow.
+     * The `zDepth` of the underlying `Paper` component.
```

Comment:
Sounds good. We can loop back around and fix the variation as a low-pri after the other docs issues are sorted.

You're making great progress!